packages/layout/src/lib/header/README.md

```diff
@@ -1,6 +1,10 @@
 # @acpaas-ui/ngx-components/layout
 
-Description goes here.
+A header component with logo that can hide itself when scrolling.
+
+## Dependencies
+- @jsprds/headroom.ts
```

Comment:
Moeten we die wel vermelden? Men zal denken dat ze die dependency ook moeten installeren in hun project.